mdtraj/formats/dcd/dcd.pyx

```diff
@@ -170,15 +170,15 @@ cdef class DCDTrajectoryFile:
     >>> f.close()
 
     >>> # read all of the data with automatic closing of the file
-    >>> with DCDTrajectoryFile('mytrajectory.dcd') as f:
```

Comment:
Actually, I'm not sure if this line was purposefully written like this or not...